source/docs/essentials/create-a-ui-component.md

```diff
@@ -0,0 +1,307 @@
+---
+id: create-a-ui-component
+title: Create a UI Component
+---
+
+In Front Commerce we split our components in two categories: the **UI**
+components available in the `theme/components` folder, and the business
+components available in the `theme/modules` and `theme/pages` folders.
+
+<blockquote class="info">
+If you feel the need to understand why we went for this organization, feel
+free to refer to [React components structure](#TODO)
+first.
+</blockquote>
+
+In this documentation, you will learn how to build a UI Component. We will do so by
+creating our own. Front-Commerce’s core UI components follow the same principles
+and you could dive into the `node_modules/front-commerce/src/web/theme/components`
+folder to find examples into our source code.
+
+But first, let's define what is an ideal UI Component.
+
+## The ideal UI Component
+
+In Front-Commerce we call UI component any component that is:
+
+- **Reusable in many contexts** If a component is used only once in the whole
+  application, it might be the sign that it does not exist purely for UI
```

Comment:
```suggestion
  application, it might be a smell that it does not exist purely for UI
```

I am not sure about this one, but as a developer « smell » gives me a direct context and more information that this is technical debt or incorrect abstraction.